app/policies/privacy_policy_policy.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class PrivacyPolicyPolicy < ApplicationPolicy
```

Comment:
Is this used anywhere? Also, the semantics seem off here, I think this should be `create?` for PrivacyPolicy::Acceptance, and the check should be `user == record.user`